pkg/sqlite/load.go

```diff
@@ -35,7 +35,7 @@ func NewSQLLiteLoader(outFilename string) (*SQLLoader, error) {
 	CREATE TABLE IF NOT EXISTS package (
 		name TEXT PRIMARY KEY,
 		default_channel TEXT,
-		FOREIGN KEY(default_channel) REFERENCES channel(name)
+		FOREIGN KEY(name, default_channel) REFERENCES channel(package_name,name)
```

Comment:
referencing a field that's part of a compound primary key requires referencing the full primary key - this was the main issue.